test/unittests/analysis_test.cpp

```diff
@@ -11,33 +11,24 @@
 using namespace evmone;
 
 constexpr auto rev = EVMC_BYZANTIUM;
-
-const auto fake_fn_table = []() noexcept
-{
-    evmone::exec_fn_table fns;
-    for (size_t i = 0; i < fns.size(); ++i)
-        fns[i] = (evmone::exec_fn)i;
-    return fns;
-}
-();
-
+const auto& op_table = get_op_table(rev);
```

Comment:
Danger of static init fiasco? in case `op_tables` in instructions.cpp is initialized after this

Reply:
The problem do not apply to `constexpr` globals, but still I can move it inside the function.

Reply:
Ah right, haven't noticed `constexpr`

Reply:
Good notice anyway.